src/envoy/utils/mixer_control.cc

```diff
@@ -17,9 +17,18 @@
 #include "src/envoy/utils/grpc_transport.h"
 
 using ::istio::mixerclient::Statistics;
+using ::istio::utils::AttributeName;
+using ::istio::utils::LocalAttributes;
+using ::istio::utils::LocalNode;
 
 namespace Envoy {
 namespace Utils {
+
+const char NodeKey::kName[] = "NODE_NAME";
```

Comment:
Can we just have UID and namespace? We don't know fully qualified UID yet, not until multi-cluster support lands in Mixer. Until then, we should treat UID as opaque, and not assume it include kubernetes:// and only has two parts.